webbpsf/roman.py

```diff
@@ -310,7 +309,7 @@ def calc_psf(
         self.options['crop_psf'] = crop_psf
 
         # add_distortion keyword is not implemented for RomanCoronagraph Class
-        if self.name == 'RomanCoronagraph' and add_distortion == True:
+        if self.name == 'RomanCoronagraph' and add_distortion is True:
```

Comment:
This could be further simplified I think to just `and add_distortion`, yes? The comparison to True can be implicit.

Reply:
Thats correct, I kept it intentionally with this check for code consistency.